MockKit/MockPumpManager.swift

```diff
@@ -296,34 +296,15 @@ public final class MockPumpManager: TestingPumpManager {
     private var stateObservers = WeakSynchronizedSet<MockPumpManagerStateObserver>()
 
     public init() {
-        let deliverableIncrements: MockPumpManagerState.DeliverableIncrements = .medtronicX22
-        state = MockPumpManagerState(
-            deliverableIncrements: deliverableIncrements,
-            supportedBolusVolumes: deliverableIncrements.supportedBolusVolumes ?? [],
-            supportedBasalRates: deliverableIncrements.supportedBasalRates ?? [],
-            reservoirUnitsRemaining: MockPumpManager.pumpReservoirCapacity,
-            tempBasalEnactmentShouldError: false,
-            bolusEnactmentShouldError: false,
-            bolusCancelShouldError: false,
-            deliverySuspensionShouldError: false,
-            deliveryResumptionShouldError: false,
-            deliveryCommandsShouldTriggerUncertainDelivery: false,
-            maximumBolus: 25.0,
-            maximumBasalRatePerHour: 5.0,
-            suspendState: .resumed(Date()),
-            pumpBatteryChargeRemaining: 1,
-            unfinalizedBolus: nil,
-            unfinalizedTempBasal: nil,
-            finalizedDoses: [],
-            progressWarningThresholdPercentValue: 0.75,
-            progressCriticalThresholdPercentValue: 0.9)
+        state = MockPumpManagerState(reservoirUnitsRemaining: MockPumpManager.pumpReservoirCapacity)
```

Comment:
[nit] As below, consider adding this as a default for `reservoirUnitsRemaining`.